app/(home)/page.tsx

```diff
@@ -69,9 +71,9 @@ function DataCollectionContent({ instance }: { instance: string }) {
   }
 
   if (!data || error) {
-    // TODO: Return error page
-    console.log('Error', error);
-    return <h1>Something went wrong</h1>;
+    return (
+      <ErrorComponent message="Unable to load Data collection. Please try again." />
```

Comment:
How do you feel about this error message? We could use this in all places where we have a larger unknown error that stops the page or components from rendering

> We're sorry, but something went wrong. Please try refreshing the page. If that doesn't help, please get in touch with us. We're monitoring the situation and working to resolve it as quickly as possible.